bgpd/bgp_btoa.c

```diff
@@ -217,6 +217,11 @@ int main(int argc, char **argv)
 
 		printf("len: %zd\n", len);
 
+		if (len > ((size_t)-1) - 12 || stream_get_size(s) < len + 12) {
```

Comment:
I don't understand why we're checking if `len` is larger than the maximum value of `size_t` less 12? This deserves an explanatory comment imo.

Reply:
It is wrong, my fault. I should check with (uint32_t)-1 instead of (size_t)-1 (because the stream_getl() function returns uint32_t and not size_t. I could add a comment about integer overflow handling if you agree on that.